weather.go

```diff
@@ -122,7 +122,7 @@ func ParamsToEsQuery(site string, datumName string, start string, end string, ag
 }
 
 func EsSearch(SearchString []byte, agg bool) (EsResponse, error) {
-	esUrl := "http://elasticsearch.lco.gtn:9200/mysql-telemetry-*/_search?pretty"
```

Comment:
While we're in here we should update the function name from `EsSearch` to `OsSearch` and the variable to `osUrl`. Also the URL we're using is `https://opensearch.lco.global/`

Reply:
updated the variable names and url